cime_config/testmods_dirs/allactive/def_cmcc_83l/shell_commands

```diff
@@ -0,0 +1,17 @@
+./xmlchange RUN_STARTDATE=0001-01-01
```

Comment:
We performed some tests in the previous configuration, and having the ocean on its own pes did not increase the performance of the simulation since the atmosphere is the slowest component in this configuration.
By the way, I should have included this PES in the config_pes, and I should have removed the NTASKS changes from here.
Are the PFS tests a test option like ERR?